plugins/quay/tests/quay.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { expect, Page, test } from '@playwright/test';
+
+test.describe('Quay plugin', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('http://localhost:3000/quay');
+    await expect(
+      page.getByRole('link', { name: 'backstage-test/test-images' }),
+    ).toBeEnabled({ timeout: 20000 });
+  });
+
+  test.afterAll(async ({ browser }) => {
+    await browser.close();
+  });
+
+  test('All columns are shown', async () => {
+    const columns = [
+      'Tag',
+      'Last Modified',
+      'Security Scan',
+      'Size',
+      'Expires',
+      'Manifest',
+    ];
+    const thead = page.locator('thead');
+
+    for (const col of columns) {
+      await expect(thead.getByText(col)).toBeVisible();
+    }
+  });
+
+  test('Vulnerabilities are listed', async () => {
+    const severity = ['High:', 'Medium:', 'Low:'];
```

Comment:
how is the data being fetched, will it change with time are always have these severities?

Reply:
it is being fetched from a quay repo that I've made for this reason. The way I see it, if the image doesn't get updated, the number of vulnerabilities should not decrease. As far as the test cares, the number just needs to stay above zero.

Reply:
yeah, it doesn't change with time then it should be good.